tests/devapps/WebApi/readme_perf.txt

```diff
@@ -81,12 +90,24 @@ Static Wilson LRU cache with normal size limitation (max size ~2GB or 700k token
      95%    18.00ms   17.00ms
      99%   116.00ms   112.00ms
 
-
-
-
-To run the perf tests, start this webapi, install a tool like https://github.com/codesenberg/bombardier and hit it with:
-
-bombardier -d 60s -l https://localhost:44355/FlatCache
-bombardier -d 60s -l https://localhost:44355/StaticDictionary
-bombardier -d 60s -l https://localhost:44355/Singleton
-bombardier -d 60s -l https://localhost:44355/WilsonLruCache
+OBO performance tests for refresh flow. Added with 4.33 release.
```

Comment:
Can you please describe the methodology used? I.e. how the cache is hit, how often the Refresh Token flow is performed (roughly), the timeout values used etc.